postprocessing/README.md

```diff
@@ -9,18 +9,30 @@ To install the package, we recommend using `uv` for Python package management.
 ### Set up uv
 
 1. Install `uv` if you don't have it already:
+
    ```bash
    wget -qO- https://astral.sh/uv/install.sh | sh
```

Comment:
Need to make sure that these instructions work on Windows and Mac. Probably need to have people build a Conda environment, then install UV inside that. Something like this: https://medium.com/@datagumshoe/using-uv-and-conda-together-effectively-a-fast-flexible-workflow-d046aff622f0